src/index.ts

```diff
@@ -0,0 +1,163 @@
+      // 'Sai Wan Ho (looking towards the east)': `https://www.hko.gov.hk/wxinfo/aws/hko_mica/swh/latest_HD_SWH.jpg`,
+    };
+    const inputMediaPhotos = _.map(photos, (url: string, location: string) => ({
+      type: 'photo',
+      media: `${url}?v=${event.scheduledTime}`,
+      caption: `${location}\n<pre>${now}</pre>`,
+      parse_mode: 'HTML',
+    }));
+    const url = `https://api.telegram.org/bot${
+      env.API_KEY
+    }/sendMediaGroup?chat_id=${
+      env.TELEGRAM_CRON_JOB_CHAT_ID
+    }&media=${JSON.stringify(inputMediaPhotos)}&disable_notification=true`;
+    console.log(222, url);
+    const result = await fetch(url);
+    console.log(333, result);
+    // await logger.info(`Scheduled job from <code>${event.cron}</code> at ${now}.\n${photo}`);
+  },
+  async fetch(
+    request: WorkerRequest,
+    env: Record<string, string>,
+    ctx: ExecutionContext,
+  ): Promise<Response> {
+    return handleRequest(request, env);
+  },
+};
+
+async function handleRequest(request: WorkerRequest, env: Env) {
+  const logger = new Logger(env);
+  try {
+    if (request.method === 'POST') {
+      const payload = await request.json();
+      if (payload?.message?.from && !payload.message.from.is_bot) {
+        const kvKey = `conversation-with-${payload.message.from.username}`;
+        const past = await env.KV.get(kvKey, { type: 'json' });
+        const kvValue = { ...past };
+        kvValue[payload.message.date] = kvValue.hasOwnProperty(
+          payload.message.date,
+        )
+          ? [...kvValue[payload.message.date], payload.message.text]
+          : (kvValue[payload.message.date] = [payload.message.text]);
+        await env.KV.put(kvKey, JSON.stringify(kvValue));
+
+        if (!payload.message.from.is_bot) {
+          const chatId = payload.message.chat.id;
+          const text = `Hi ${payload.message.from.first_name} ${
+            payload.message.from.last_name
+          } @${
+            payload.message.from.username
+          }\nyour request payload is<pre><code class="language-json">${JSON.stringify(
+            payload,
+            null,
+            2,
+          )}</code></pre>`;
+
+          const url = `https://api.telegram.org/bot${env.API_KEY}/sendMessage?chat_id=${chatId}&text=${text}&parse_mode=HTML&reply_to_message_id=${payload.message.message_id}`;
+          await fetch(url);
+          await logger.info(
+            `Received msg from @${
+              payload.message.from.username
+            }.\nPayload:<pre><code class="language-json">${JSON.stringify(
+              payload,
+              null,
+              2,
+            )}</code></pre>`,
+          );
+        }
+      }
+    }
+
+    return new Response(dayjs().toISOString()); // Doesn't really matter
+  } catch (exception) {
+    if (exception instanceof Error) {
+      await logger.error(`${exception.stack}`, request);
+    }
+
+    return new Response('NOT OK'); // Doesn't really matter
+  }
+}
```

Comment:
What about move to a new file?
```
\src
   \features
      fetchEventHandler.ts
      scheduledEventHandler.ts
   \infras
      logger.ts
   index.ts
```

Reply:
So we can move these to `fetchEventHandler.ts`

Reply:
Done